### ISSUE TITLE
resetOptions and options.crypto

### ISSUE THREAD

**Issue by sm2017** [yeojz/otplib#136]
**Describe the bug**
 we have "Expecting options.crypto to have a createHmac function" error , when we resetOptions

**To Reproduce**
run the code

```js
var otplib = require("otplib")
otplib.totp.resetOptions();
const secret = 'KVKFKRCPNZQUYMLXOVYDSQKJKZDTSRLD'
const isValid = otplib.authenticator.check('12345', secret);
```

**Expected behavior**
If you comment `otplib.totp.resetOptions();` we have no issue , but resetOptions don't reset truly an we have "Expecting options.crypto to have a createHmac function" error

**Environment (please provide any relevant information):**
 - OS: Windows
 - Browser:---
 - Node / Yarn: v10.14.2  /  1.13.0
 - Any frameworks: No

**Comment by sm2017** [comment]
@yeojz Can you reply please?

**Comment by yeojz** [comment]
Beta build: `otplib@10.2.0-0`

**Comment by yeojz** [comment]
released in v10.2.2
